app/src/main/java/com/jalasoft/routesapp/data/remote/managers/FirebaseManager.kt

```diff
@@ -47,4 +48,13 @@ class FirebaseManager(val db: FirebaseFirestore) : FirebaseDataSource {
             Response.Error(e.message.toString(), null)
         }
     }
+
+    suspend inline fun <reified T : Any> getDocumentsByReferenceWithCondition(collection: FirebaseCollections, field: String, parameter: DocumentReference): Response<List<T>> {
+        return try {
```

Comment:
The previous function does the same I think, give it a check

Reply:
no it doesn't work because i'm sending a documentReference as parameter